README.md

```diff
@@ -14,6 +14,8 @@ DGL is an easy-to-use, high performance and scalable Python package for deep lea
   <b>Figure</b>: DGL Overall Architecture
 </p>
 
+## ![new](https://i.imgur.com/1NilWUd.png =40x)DGL News
```

Comment:
Better not to use imgur because it's blocked in mainland China. Consider using our s3 bucket.